airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_check.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env bash
 set -o errexit -o nounset -o pipefail
 
-# Check if airbyte-ci is on the PATH and pointing to the correct location
+echo "Checking if airbyte-ci is correctly installed..."
+
+# Check that airbyte-ci is on the PATH
+# If not print an error message and exit
+if ! which airbyte-ci >/dev/null 2>&1; then
+    echo "airbyte-ci is not on the PATH"
+    echo "Check that \$HOME/.local/bin is part of the PATH"
```

Comment:
I'm not sure I get what's exactly checking that `\$HOME/.local/bin is part of the PATH`
Could we have a prior step that just checks:
* `~/.local/bin` directory exists 
* That this path is part of the `$PATH` variable?

Reply:
Good catch. I had the absense of airbyte-ci as an indication of a improper path.

Update these to be separate checks